db/pre-re/item_db.txt

```diff
@@ -4829,7 +4829,7 @@
 12261,Secret_Medicine,Leap of Fantasy,2,0,,0,,,,,0xFFFFFFFF,7,2,,,,,,{ getexp2(2000000,1000000); },{},{}
 12262,Inspector_Certificate_,Authoritative Badge,2,0,,10,,,,,0xFFFFFFFF,7,2,,,,,,{ sc_start SC_SPEEDUP0,540000,25; },{},{}
 12263,Comp_Battle_Manual,Field Manual,2,2,,10,,,,,0xFFFFFFFF,7,2,,,,,,{ sc_start SC_EXPBOOST,1800000,50; },{},{}
-12264,Comp_Bubble_Gum,Bubble Gum,2,2,,10,,,,,0xFFFFFFFF,7,2,,,,,,{ sc_start SC_ITEMBOOST,1800000,200; },{},{}
+12264,Comp_Bubble_Gum,Bubble Gum,2,2,,10,,,,,0xFFFFFFFF,7,2,,,,,,{ sc_start SC_ITEMBOOST,1800000,50; },{},{}
```

Comment:
Shouldn't this be 100? Just curious where 50 came from. The description states:
```
30분간 몬스터가 아이템을 드롭할 확률이 100% 증가한다.
```

Reply:
Unfortunately, this PR is more than a year old. I don't remember but I have to guess the item effect changed meanwhile or was mistranslated at the time I made this. Change it to what the correct number currently is please.